src/flow/dinitz.jl

```diff
@@ -0,0 +1,141 @@
+"""dinitz_impl
+Computes the maximum flow between the source and target vertexes in a flow
```

Comment:
You don't need the function name in the docstring. Just start with "Computes..."